test/language/statements/class/static-init-scope-lex-close.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-runtime-semantics-classstaticblockdefinitionevaluation
+description: Destruction of environment record for variable-scoped bindings
+info: |
+  ClassStaticBlock : static { ClassStaticBlockBody }
+
+    1. Let lex be the running execution context's LexicalEnvironment.
+    2. Let privateScope be the running execution context's PrivateEnvironment.
+    3. Let body be OrdinaryFunctionCreate(Method, « », ClassStaticBlockBody, lex, privateScope).
+features: [class-static-block]
+---*/
+
+var test262 = 'outer scope';
+var probe;
+
+class C {
+  static {
+    let test262 = 'first block';
+  }
+  static {
+    probe = test262;
```

Comment:
I couldn't figure out how we're using probe - should there be an assertion for ti?